content/_data/upgrades/2-235-3.adoc

```diff
@@ -24,3 +24,46 @@ Update Red Hat compatible operating systems (Red Hat Enterprise Linux, CentOS, F
 ----
 
 See the link:/blog/2020/07/27/repository-signing-keys-changing/[blog post] for more details.
+
+==== New Windows Installer
+
+Beginning with Jenkins LTS release link:/changelog-stable/#v2.235.3[2.235.3], the same Windows installers is used in the stable release as has been used since April 2020 in the weekly release.
+The new Windows installer was announced in a link:/blog/2019/02/01/windows-installers/[blog post].
+
+Jenkins administrators on Windows may choose to continue using and upgrading their existing installation or they may choose to migrate to the new Windows installer.
+
+===== Upgrade with existing installation
+
+Jenkins administrators on Windows are offered new Jenkins versions from within the "Manage Jenkins" page of their installation.
+After the initial installation of Jenkins on Windows, administrators perform upgrades through the "Manage Jenkins" page.
+Administrators generally do not use the Jenkins MSI installer to perform a Jenkins upgrade because the upgrade is simple and easy from within Jenkins itself.
+
+Administrators that choose to continue using their existing installation will continue to be able to upgrade to new Jenkins versions.
+Settings in the Jenkins installation can be modified by editing the `jenkins.exe.config.xml` file and other configuration files.
+
+In this configuration, Jenkins will continue to use the 32 bit Java Runtime Environment that was included with the previous installer.
```

Comment:
```suggestion
In this configuration, Jenkins will continue to use the 32 bit Java Runtime Environment that was included with the previous installer, unless they manually modify the jenkins.exe.config to point to a 64 bit Java Runtime Environment.
```

Reply:
It may be useful referencing a valid 64 bit Java Runtime Environment configuration inside jenkins.exe.config file.
Content of jenkins.exe.config file, generated by Jenkins installer 2.235.2, doesn't contain explicit reference to 32 bit Java Runtime Environment. For this reason it's not obvious which fields should be modified for pointing to a 64 bit Java Runtime Environment.